### PR TITLE
Add user class and test

### PR DESCRIPTION
**Pull request by cagallo**
#### What's this PR do?

- Start users.js and the test for the file

#### Where should the reviewer start?

- User.js, User-test.js

#### How should this be manually tested?

- N/A

#### Any background context you want to provide?

- N/A

#### What are the relevant tickets?

- N/A

### CODE REVIEW

**Comment by kammon10** [comment]
good start!

File: src/classes/User.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class User {
```
Comment (anewb87) [inline src/classes/User.js:1]: Starting to build out the skeleton.

Comment (aliroemhildt) [inline src/classes/User.js:1]: this looks like a good start

**Review by aliroemhildt — COMMENTED** [review]
Nice start!

**Review by anewb87 — COMMENTED** [review]
Starting strong.

**Pull request merged**